uwp/Charts/Series.md

```diff
@@ -138,7 +138,7 @@ chart.Series.Add(series2);
 
 {% endtabs %}
 
-![Colum SegmentSpacing support in UWP Chart](Series_images/SegmentSpacing.png)
+![Colum SegmentSpacing support in UWP Chart](series_images/segmentspacing.png)
```

Comment:
You make the UWP/Chart/Series.md file changes in the below PR too.
https://github.com/syncfusion-content/uwp-docs/pull/371/files
Which one is Final?

Reply:
Seems both PR's are duplicate. Pleas check and share the Final PR.